fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -938,7 +938,7 @@ public CommandProcessingResult makeGLIMLoanRepayment(final Long loanId, final Js
 
         final Long parentLoanId = loanId;
 
-        glimRepository.findById(parentLoanId).get();
+        glimRepository.findById(parentLoanId).orElse(null);
```

Comment:
This is not gonna fly. This rather needs to be an existence check.
If you wanna keep it that way, extract the result it to a variable and do a null-check to make sure the code fails in case the parent loan doesn't exist.